### PR TITLE
feat(command): add get-roles command

### PR DESCRIPTION
**Pull request by dclayton-godaddy**
This adds a new `get-roles` command used to get all the AWS roles which can be used to write AWS profiles or output in any format. See <README.rst> for details.

```bash
# outputs all roles as AWS profiles and save in ~/.aws/credentials
aws-okta-process get-roles -u jdoe -o mycompany.okta.com --output=profiles > ~/.aws/credentials
```

### CODE REVIEW

File: aws_okta_processor/cli.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    try:
+        argv = [args['<command>']] + args['<args>']
+        if args['<command>'] == 'authenticate':
+            options = docopt(commands.authenticate.__doc__, argv=argv)
+            command = commands.authenticate.Authenticate(options)
+            command.run()
+        elif args['<command>'] == 'get-roles':
+            options = docopt(commands.get_roles.__doc__, argv=argv)
+            command = commands.get_roles.GetRolesCommand(options)
```
Comment (jeffpearce) [inline aws_okta_processor/cli.py:40]: Nit: I wish this used the same naming pattern as `Authenticate`, which doesn't include the suffix `Command`. In this case `commands.get_roles.GetRoles` would match the pattern

Comment (jeffpearce) [reply aws_okta_processor/cli.py:40]: Why is there a mix of snake_case and PascalCase naming?

Comment (azweb76) [reply aws_okta_processor/cli.py:40]: I prefer a class to be a noun.

Comment (jeffpearce) [reply aws_okta_processor/cli.py:40]: Then Authenticate is the odd person out :-P

Comment (jeffpearce) [reply aws_okta_processor/cli.py:40]: Ah, I misunderstood the code

Comment (azweb76) [reply aws_okta_processor/cli.py:40]: Ya. I can change that.

Comment (azweb76) [reply aws_okta_processor/cli.py:40]: Obviously that is a breaking change.

Comment (dclayton-godaddy) [reply aws_okta_processor/cli.py]: Renamed `GetRolesCommand` to `GetRoles` to avoid breaking change.

**Review by jeffpearce — APPROVED** [review]
💯

**Review by tahoward — CHANGES_REQUESTED** [review]
Thank you, this will be a great contribution! I ran this locally and found a couple edge cases that should get addressed:

1. Running `aws-okta-processor get-roles` prompts for missing options but the response returns `null` for org and user attributes.
2. Running `aws-okta-processor get-roles` twice results in an error for second command:
```
Info: Calling https://None/api/v1/sessions/me/lifecycle/refresh
Error: Connection Error
```

**Review by tahoward — APPROVED** [review]
Approved

**Pull request merged**